src/components/note/Note.tsx

```diff
@@ -27,15 +30,21 @@ const Note = ({
   handleEditNote,
   filterNotes,
   categories: categoriesArray,
+  handleDeleteAttachment,
+  handleSaveAttachment,
 }: INote) => {
   const [displayModal, { setFalse: handleClose, setTrue: openModal }] = useBoolean(false)
   const [displayRemoveModal, { setFalse: closeRemoveModal, setTrue: openRemoveModal }] = useBoolean(false)
   const [displayEditModal, { setFalse: closeEditModal, setTrue: openEditModal }] = useBoolean(false)
```

Comment:
Semantyka tego hooka useBoolean ;d

Reply:
Rozwiń myśl, bo nie wiem czy nie śmieszkujesz teraz ze mnie 😆

Reply:
Nie śmieszkuje :D Ale na pierwszy rzut oka nie miałbym pojęcia do czego służy ten hook trzeba by go było lepiej nazwać ;D

Reply:
Jakaś propozycja?

Reply:
Bo jego głowne zdanie to ustawiac true / false

Reply:
Imho w ogóle nie potrzebny hook. Bo na dobrą sprawę możesz po prostu użyć useState'a i tak i tak będzie tutaj powtarzalność. Ale widzę, że wykorzystujesz go głównie przy modalach, to równie dobrze mógłbyś zrobić po prostu całego customHooka na wyświetlanie modala, który zwraca jego dostępne wartości.

Reply:
A i typescripta, przerzuć do jednego exportowanego pliku w stylu typescript.ts. Tam wrzuć interfejsy i je eksportuj, łatwiej odnaleźć się w takim pliku, bo tak widziałem, że bodajże w kilku przypadkach do innych interfejsów w pliku, zgarniasz jeszcze inne z jeszcze innych plików. A tak to masz ogólny jeden plik z typami i je eksportujesz :D

Reply:
typy i interfejsy przerzucę już w osobnym issue

Reply:
i chyba tak samo przekminie z modalam i useBoolean

Reply:
ogólnie można kiedyś zrobić zdzwonkę i przelecieć przez projekt i rzucisz uwagami, jak będziesz mieć chwilę czasu

Reply:
luzik, normalnie jak coś to pisz na dc czy coś :D